src/vm/wren_core.wren

```diff
@@ -302,6 +302,52 @@ class List is Sequence {
     }
     return result
   }
+
+  sort() { sort_(this.map {|n| [n, n] }.toList) }
+
+  sort(key) { sort_(this.map {|n| [n, key.call(n)] }.toList) }
```

Comment:
Why did you do this? Did your performance measurements show that just the function call for each comparison is likely to have more overhead than the duplication of the entire list into pairs and later duplicating it back to single elements?